### PR TITLE
Adds inline-size to horizontal tabs wrapper.

### PR DESCRIPTION
**Pull request by Ben-Pusey-Bentley**
## Changes

Adds `inline-size: 100%` to horizontal tabs wrappers. This change is meant to counteract an issue caused by this [PR](https://github.com/iTwin/iTwinUI/pull/1884), where adding `contain: inline-size` caused the tabs wrapper to change it's implicit width.

## Testing

Ran css and react visual tests to make sure that there are no unintended visual changes.

## Docs

Added patch changesets to both the react and css packages.

### CODE REVIEW

File: packages/itwinui-css/src/tabs/base.scss [hunk 1]
```diff
@@ -130,6 +130,7 @@ $borderless-horizontal-tab-min-height: calc(
   grid-template-columns: 1fr auto;
   grid-template-rows: auto 1fr;
   contain: inline-size;
+  inline-size: 100%;
```
Comment (mayank99) [inline packages/itwinui-css/src/tabs/base.scss:133]: You can undo #1911 now

File: .changeset/great-news-beg.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Adds `inline-size` to `Tabs.Wrapper` to counteract visual change to the width of horizontal tabs wrappers.
```
Comment (mayank99) [inline .changeset/great-news-beg.md:5]: Changesets should be worded in a way that emphasizes why the user would care about this change.

```suggestion
Fixed a regression in `Tabs` where the panel content was not occupying the full width of the container.
```

File: packages/itwinui-css/src/tabs/base.scss (hunk 1 above)
Comment (Ben-Pusey-Bentley) [reply packages/itwinui-css/src/tabs/base.scss:133]: Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1955/commits/7f88dcd263e6519b38730bca28f5d150a2ca0483)

**Review by mayank99 — APPROVED** [review]

File: .changeset/great-news-beg.md (hunk 2 above)
Comment (Ben-Pusey-Bentley) [reply .changeset/great-news-beg.md]: Changed along with the css changeset in this [commit.](https://github.com/iTwin/iTwinUI/pull/1955/commits/8177c00df7a68b47737f9f2b92017bb2ecb84aee)

**Pull request merged**